go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/splunk/splunk-operator
 go 1.13
 
 require (
-	github.com/aws/aws-sdk-go v1.17.7
+	github.com/aws/aws-sdk-go v1.34.1
```

Comment:
I see this particular version v1.34.1 of library was last released in 2020-08-10, while the latest version is v1.37.21 (released today!). What would some of the primary reasons be to stick to an older version?

Reply:
I don't have any objections going to the newest version, I was mainly worried about any compatibility issues as we experienced with the newer versions of some other libraries (caused build failures). I have pushed a commit to update to v1.37.21 and will check the CI results.